src/components/SwitchButton/index.tsx

```diff
@@ -80,11 +102,47 @@ const SwitchButton = ({
     setLocalValue(event.target.value)
   }
 
+  useEffect(() => {
+    if (!leftButtonRef.current || !rightButtonRef.current) return
+    setLeftCardWidth(leftButtonRef.current.getBoundingClientRect().width)
+    setRightCardWidth(rightButtonRef.current.getBoundingClientRect().width)
+  }, [leftButton.value, leftButtonRef, localValue, rightButtonRef])
+
+  const setMouseDown = (isMouseDown: boolean) => () =>
+    setHasMouseDown(isMouseDown)
+
+  const getScaleValue = () => {
+    if (!hasMouseDown) return 1
+    const offsetScaleRatio = 6
+    const currentWidth =
+      localValue === leftButton.value ? leftCardWidth : rightCardWidth
+
+    return 1 + offsetScaleRatio / currentWidth
+  }
+
   return (
     <Tooltip text={tooltip}>
       <div style={{ display: 'inline-flex' }}>
-        <StyledBorderedBox>
+        <StyledBorderedBox
+          onMouseDown={setMouseDown(true)}
+          onMouseUp={setMouseDown(false)}
+          onMouseLeave={setMouseDown(false)}
+        >
+          <FocusOverlay
+            data-focusposition={
+              localValue === leftButton.value ? 'left' : 'right'
+            }
+            style={{
+              transform: `translate3d(${
+                localValue === leftButton.value ? 0 : leftCardWidth + 8
```

Comment:
Where does the `8` come from?  I would prefer to have it in a top-level const with a name to understand what it is.